ncm-spma/src/main/pan/components/spma/dnf/schema.pan

```diff
@@ -22,4 +22,14 @@ type component_spma_dnf = {
     "run"                ? boolean  # Run the transaction after configuring DNF
 };
 
+@{DNF module structure. To be used as dict, with name as key.
+ to set default profiles for modules not enabled by default and change existing defaults.
+}
+type component_spma_dnf_module_defaults = {
+    'stream'   : string
+    'profiles' ?  string[]
+    @documentation{ defaults to 1 in component, provided in-case it changes in a future release }
+    'modulemd_version' : long = 1
```

Comment:
Looks like [there is already a version 2](https://github.com/fedora-modularity/libmodulemd/blob/main/yaml_specs/modulemd_stream_v2.yaml), we should probably at least put a lower bound on this, but maybe just restrict it to `1` until support is implemented for another version?
```suggestion
    'modulemd_version' : long(1..) = 1
```